lib/jobs/ipmi-catalog.js

```diff
@@ -110,11 +110,26 @@ function ipmiCatalogJobFactory(
         assert.string(command);
         assert.object(obmSettings);
 
-        return [ 
+        // Some commands end with " || true"
+        // This tail needs to be deleted from the command args
+        // Otherwise the tail passed to ipmitool child process will cause error response
+        // this tail is stored and used to determine whether continue sending other commands
+        var ignore = false;
+        var tailIdx = command.indexOf(' ||');
```

Comment:
Didn't realize remove the "|| true" will cause not able to find the command parser, so I think this is not a good design for looking up command parser, we need improvement for it.

If "|| true" is a kind of flag to ignore failure, I would prefer to explicitly define a flag for each command to indicate whether ignore error.

Considering the urgency of  1.1.x release, I agree to merge into 1.1.x branch, but we can improve this in master.